auth/service.go

```diff
@@ -142,33 +165,33 @@ func (s *Service) getOAuthToken(ctx context.Context) (*string, error) {
 
 	validationerror := ValidateResponse(ctx, client, res)
```

Comment:
variable here could be simply named `err` (not a required change)

Reply:
well, not changed in this PR, but why not... 
renamed in 731ab69